.stylelintrc.yaml

```diff
@@ -0,0 +1,29 @@
+extends:
+  - stylelint-config-standard
+  - stylelint-config-rational-order
+plugins:
+  - stylelint-order
+  - stylelint-config-rational-order/plugin
+rules:
+  # Override stylelint-config-standard
+  rule-empty-line-before:
+  - always
+  - except:
+    - after-single-line-comment
+    - first-nested
+  # Single-quotes are our convention throughout our codebase within CSS
+  string-quotes: single
+  # Follow best practices
+  font-family-name-quotes: always-unless-keyword
+  font-weight-notation: numeric
+  selector-attribute-quotes: always
+  color-hex-length: long # TODO: Think about using only RGBA notation
```

Comment:
HEX format for color values is obsolete, IMO. RGB/RGBA and HSL are more useful, especially with CSS custom properties.